tests/integration/revocation.test.js

```diff
@@ -156,6 +156,18 @@ describe('Revocation Module', () => {
     const transaction = dock.revocation.newRegistry(registryID, policyNew, false);
     await expect(dock.sendTransaction(transaction)).resolves.toBeDefined();
     const reg = await dock.revocation.getRevocationRegistry(registryID);
-    expect(!!reg).toBe(true);
+    const controllerSet = reg.policy._raw;
+    expect(controllerSet.size).toBe(2);
+
+    let hasFirstDID = false;
+    let hasSecondDID = false;
+    controllerSet.forEach(controller => {
+      if (controller.toString() === controllerDID) {
+        hasFirstDID = true;
+      } else if (controller.toString() === controllerDIDTwo) {
+        hasSecondDID = true;
+      }
+    });
+    expect(hasFirstDID && hasSecondDID).toBe(true);
```

Comment:
Is this equivalent to checking `new Set(controllerSet) === controllersNew`?

Reply:
Maybe more like 
`new Set(controllerSet.map(e => e.toString())) === controllersNew`

Reply:
it is equiv, because you cant do `===` check on a Set/array. perhaps it could be written cleaner though https://stackoverflow.com/questions/31128855/comparing-ecma6-sets-for-equality